src/app.ts

```diff
@@ -3,8 +3,8 @@ import { set } from '@balena/es-version';
 set('es2022');
 
 import { setDefaultAutoSelectFamilyAttemptTimeout } from 'net';
-// Increase the timeout for the happy eyeballs algorithm to 5000ms to avoid issues on slower networks
-setDefaultAutoSelectFamilyAttemptTimeout(5000);
+// Increase the timeout for the happy eyeballs algorithm to 3000ms to avoid issues on slower networks
+setDefaultAutoSelectFamilyAttemptTimeout(3000);
```

Comment:
@pipex you seem to have updated the PR title to 4.5s but this is still 3s

Reply:
Oops, forgot to push